params/config.go

```diff
@@ -639,6 +639,7 @@ type ChainConfig struct {
 	DarwinTime          *uint64  `json:"darwinTime,omitempty"`          // Darwin switch time (nil = no fork, 0 = already on darwin)
 	DarwinV2Time        *uint64  `json:"darwinv2Time,omitempty"`        // DarwinV2 switch time (nil = no fork, 0 = already on darwinv2)
 	EuclidTime          *uint64  `json:"euclidTime,omitempty"`          // Euclid switch time (nil = no fork, 0 = already on euclid)
+	EuclidV2Time        *uint64  `json:"euclidv2Time,omitempty"`        // EuclidV2 switch time (nil = no fork, 0 = already on euclidv2)
```

Comment:
Should we change `v2` to `V2` of `DarwinV2Time` and `EuclidV2Time` to make it more consistent with `genesis.json`? The config unmarshaling is not case-sensitive though.

Reply:
this is due to golang's https://pkg.go.dev/encoding/json#Unmarshal also accepting case-insensitive matching. maybe a non-issue.